lib/Mojo/WebSocketProxy/Backend/ConsumerGroups.pm

```diff
@@ -427,4 +438,38 @@ sub _prepare_request_data {
     return $msg_type, $request_data;
 }
 
+=head2 get_proper_stream_name
+
+This method looking for stream name that message should be added to,
+I<general> is the default stream name if either C<$proposed_stream_name> is not provided or C<queue_separation_enabled> is 0.
+Also C<$proposed_stream_name> will be ignored if C<queue_separation_enabled> is 0.
```

Comment:
This is just repeating part of the previous sentence?

Reply:
not exactly, working on more clear explaination